### DISCUSSION TITLE
How to use callback when PDF loaded

### DISCUSSION THREAD

**Discussion by BearXmX** [mozilla/pdf.js#18005] [Q&A]
when i use  web viewer draw PDF file and i use iframe to load,the iframe src like /web/viewer.html?file=blob

when PDF loaded, i wish the iframe can use postmessage api send some message to parent container, so how can I use lifeCycles in the 
viewer.html

Thank you very much!

**Comment by moon476607596** [comment]
Have you solved this problem？
